src/lib.rs

```diff
@@ -330,9 +330,9 @@ impl EmojiLabel {
             ui.spacing_mut().item_spacing.x = 0.0;
             match segment {
                 TextSegment::Text(text) => {
-                    let mut label = egui::Label::new(text.clone()).truncate(self.truncate);
+                    let mut label = egui::Label::new(text.clone()).truncate();
```

Comment:
This will set the `truncate` flag on the `egui::Label` even if `self.truncate` is false. The API of `EmojiLabel` needs to be changed to match the new `egui::Label` API, and this line should be fixed to only call `truncate()` if it was called on the `EmojiLabel`.